### DISCUSSION TITLE
"Encountered backfill data we don't recognise"

### DISCUSSION THREAD

**Discussion by jengleruk** [NightscoutFoundation/xDrip#3612] [General]
Hi - seeing this, and it seems the sensor (G7) has failed.

Is that message a "regular" one for a sensor failure, or have we done something wrong?

One thing that came to mind was that we left the phone in UK time even though we are now in Italy (one hour ahead).

Could that have caused an issue?

Thank you all.

![WhatsApp Image 2024-08-05 at 12 08 45_3dcd5fc2](https://github.com/user-attachments/assets/e3a36392-5ce9-4d29-a410-1f5a00e38450)
![WhatsApp Image 2024-08-05 at 12 19 20_c94498fc](https://github.com/user-attachments/assets/2b69615e-e1ab-4be2-bca1-c18584b0b954)

**Comment by Navid200** [comment]
Can you post a screenshot of the status page?

**Reply by jengleruk** [reply]
Hi, I couldn't do the above as we already replaced the sensor.

But the same things has just happened again unfortunately - in the middle of the night. Same error messages on event log (backfill data we don't recognise) and same "sensor failed 7". Here is a screenshot of the status page (both pages): 

![WhatsApp Image 2024-08-09 at 08 20 16_0bcf0f20](https://github.com/user-attachments/assets/8ae8f7ae-629f-4b2f-8d48-5f96b16d57d2)
![WhatsApp Image 2024-08-09 at 08 20 33_25ac40d2](https://github.com/user-attachments/assets/264616d1-fc21-4a5c-93af-4721bc884676)

Thank you very much.

The important question as I see it is: is this a "regular" sensor failure and something to do with the sensor (ie it would have happened using the Dexcom software), or is is a problem connected to Xdrip+ ?

**Reply by Navid200** [reply]
I don't recall having experienced this error myself over the past 10 months I have been using G7 consistently.  

This error is sent by G7.  It is not an error message created by xDrip.  

Whenever I wonder if something is caused by xDrip, I use the Dexcom app to confirm.  If something works fine with Dexcom app but, does not with xDrip, we have something to fix for sure.

**Reply by jengleruk** [reply]
Thanks.  I thought about trying the Dexcom app to see if that worked, but unfortunately I couldn't get it to work as I had uninstalled it and as we are not in our home country (nor in a country in which they sell their product) it wouldn't work after reinstalling.

If this happens when we are at home I will try it again though.

For now have started another sensor. Fingers crossed!

**Comment by royio** [comment]
I'm experiencing this sudden backfill data error also! Never experienced this error before. My sensor still has 7 days left. Hmmmmm 
![Screenshot_20240807-125612_xDrip+](https://github.com/user-attachments/assets/6f4074f2-9b8d-48df-9141-7fbb587adbd8)
![Screenshot_20240807-125740_xDrip+](https://github.com/user-attachments/assets/4060ef76-c6b6-4838-b259-d8d34845723c)

**Reply by Navid200** [reply]
You are using G7.  You are using a very old version of xDrip.

These are the instructions for G7: https://navid200.github.io/xDrip/docs/Dexcom/G7.html
